link directly to the Methodolgy & Supporting Documents page (#32)
RMI-PACTA/pacta.loanbook: vignettes/cookbook_preparatory_steps.Rmd

- [pacta.scenario.data.preparation](https://github.com/RMI-PACTA/pacta.scenario.data.preparation)
- [workflow.scenario.preparation](https://github.com/RMI-PACTA/workflow.scenario.preparation)

> Since RMI has taken over stewardship of PACTA, the prepared scenario files can also be accessed as CSV downloads on the [PACTA website](https://pacta.rmi.org/pacta-for-banks-2020/) under the ["Methodology and Documents" tab](https://pacta.rmi.org/pacta-for-banks-2020/methodology-and-supporting-materials/) of the "PACTA for Banks" section. The files are usually updated annually based on the latest scenario publications and as a general rule, the year of the publication defines the initial year of the scenario data set. This is commonly also used as the start year of the analysis.

Reviewer: ```suggestion
Since RMI has taken over stewardship of PACTA, the prepared scenario files can also be accessed as CSV downloads in the ["Methodology and Supporting Documents" section of the PACTA website](https://pacta.rmi.org/pacta-for-banks-2020/methodology-and-supporting-materials/). The files are usually updated annually based on the latest scenario publications and as a general rule, the year of the publication defines the initial year of the scenario data set. This is commonly also used as the start year of the analysis.
```
